### PR TITLE
address feedback for reports

### PR DESCRIPTION
**Pull request by samplackett**
https://eaflood.atlassian.net/browse/DOC-1370

### CODE REVIEW

File: app/views/reports-list/report-form-fields.njk [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        Select the scheme to view data for
+      </label>
+      <select class="govuk-select" id="schemeId" name="schemeId">
+        {% for scheme in schemes %}
+          <option value="{{scheme.schemeId}}">{{scheme.name}}</option>
+        {% endfor %}
+      </select>
+    </div>
+    <div class="govuk-form-group">
+      <label class="govuk-label govuk-label--s" for="year">
```
Comment (GodsonLeigh) [inline app/views/reports-list/report-form-fields.njk:16]: I assume these classes are set this way in the matching stylesheet, however the --s in GDS is -s, if that makes sense to change or not is subjective.

File: test/unit/helpers/readable-stream-return.test.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
+const { readableStreamReturn } = require('../../../app/helpers')
+
+describe('readable stream return', () => {
+  let mockResponse
+  let mockH
```
Comment (GodsonLeigh) [inline test/unit/helpers/readable-stream-return.test.js:5]: Could mockH be more descriptive?

File: test/unit/routes/schemes.test.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    const handler = routes.find(route => route.path === '/payment-schemes' && route.method === 'GET').options.handler
+    get.mockResolvedValue({
+      payload: {
+        paymentSchemes: [
+          { name: 'SFI' },
+          { name: 'Other Scheme Name' }
+        ]
+      }
+    })
+    const h = {
```
Comment (GodsonLeigh) [inline test/unit/routes/schemes.test.js:18]: could const = h be descriptive?

**Review by GodsonLeigh — COMMENTED** [review]
approve-peeling-and-peeling-away-andy-barlow

**Review by GodsonLeigh — APPROVED** [review]
approve-pr-review

File: app/constants/claim-level-report-fields.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+  PaymentCurrency: 'currency',
+  LatestFullClaimAmount: 'value',
+  LatestSitiPR: 'paymentRequestNumber',
+  LatestInDAXAmount: 'daxValue',
+  LatestInDAXPR: 'daxPaymentRequestNumber',
+  OverallStatus: 'overallStatus',
+  CrossBorderFlag: 'crossBorderFlag',
+  LatestTransactionStatus: 'status',
+  ValueStillToProcess: 'valueStillToProcess',
+  PRsStillToProcess: 'prStillToProcess'
```
[the remark this answers is not in the document]
Comment (samplackett) [reply app/constants/claim-level-report-fields.js]: Changed to PRStillToProcess

File: app/views/reports-list/report-form-fields.njk (hunk 1 above)
Comment (samplackett) [reply app/views/reports-list/report-form-fields.njk]: amended to -s and also corrected other places in the code where it was also --s

**Pull request merged**